internal/pkg_npm/pkg_npm.bzl

```diff
@@ -98,6 +98,10 @@ PKG_NPM_ATTRS = dict(NODE_CONTEXT_ATTRS, **{
         doc = """Other targets which produce files that should be included in the package, such as `rollup_bundle`""",
         allow_files = True,
     ),
+    "include_esm": attr.bool(
```

Comment:
this is great, do you know why we need this flag though? we don't have flags for the other JS flavors. Would something go wrong if we just made this always true for everyone in the next release?

Reply:
Sorry for the late reply! I don't think it would break anything so I've removed the flag.